css/src/components/steps.scss

```diff
@@ -0,0 +1,20 @@
+$steps-gap: 7px;
+$steps-size: 8px;
```

Comment:
Any chance that the size will need to scale with font size?

Reply:
I don't think so, I got these measurements from the designer and the fluent figma has a fixed size

Reply:
Was going to suggest that we do font-size scaling anyway; it's something nearly all components do and will protect us from design churn ;).

I think this will work (I've also used aspect-ratio too instead of definite heights):

```scss
$step-font-size: $font-size-7 !default;
$steps-gap: 0.4375em !default;
$steps-size: 0.5em !default;
$steps-size-double: 2 * $steps-size;

.steps {
	display: flex;
	gap: $steps-gap;
	font-size: $step-font-size;
	flex-wrap: nowrap;

	.step {
		height: $steps-size;
		aspect-ratio: 1 / 1;
		border-radius: $border-radius-rounded;
		background-color: $secondary;

		&.step-selected {
			aspect-ratio: 2 / 1;
			background-color: $primary;
		}
	}
}